kumascript/macros/LiveSampleURL.ejs

```diff
@@ -15,7 +15,7 @@ if ($1 && $1.length > 0) {
 }
 
 wiki.ensureExistence(pagePath);
-const samplePath = `${pagePath}/_sample_.${web.slugify(web.safeDecodeURIComponent($0))}.html`;
+const samplePath = `${pagePath}/_sample_.${web.slugify(web.safeDecodeURIComponent($0))}.html`.replace(/\/\/+/g, "/");
```

Comment:
🟠 Why do we need this replacement? Could we extract it to a self-describing function?

Reply:
again, bad kumascript stuff, I could be persuaded into having a `removeTrailingSlash()` on `pagePath` but I don't wanna add to much here. Extracting `.replace(/\/\/+/g, "/")` to `removeMultipleSlashes` that's only used once feels wrong.